package.json

```diff
@@ -3,18 +3,17 @@
   "version": "3.0.14",
   "description": "A modal dialog opened with a <details> button.",
   "repository": "github/details-dialog-element",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
   "scripts": {
     "clean": "rm -rf dist",
     "lint": "github-lint",
-    "prebuild": "npm run clean && npm run lint && mkdir dist",
-    "build-umd": "BABEL_ENV=umd babel index.js -o dist/index.umd.js && cp index.js.flow dist/index.umd.js.flow",
-    "build-esm": "BABEL_ENV=esm babel index.js -o dist/index.esm.js && cp index.js.flow dist/index.esm.js.flow",
-    "build": "npm run build-umd && npm run build-esm",
+    "prebuild": "npm run clean && mkdir dist",
+    "build": "tsc && cp src/index.css dist/index.css",
     "pretest": "npm run build",
-    "test": "karma start test/karma.config.js",
+    "test": "karma start test/karma.config.cjs",
```

Comment:
What's `.cjs`?

Reply:
This lets NodeJS know that this file is a `CommonJS` file - which means `import ` is a syntax error and `require`/`module` are in the global scope. Because the project's package.json has `type: module`, anything that needs to be commonjs needs a `.cjs` extension, or to be in it's own folder with a package.json that has `type: commonjs`.